demo_scripts/containerAppsPython/README.md

```diff
@@ -0,0 +1,140 @@
+# Create a Container App On Azure
+Azure Container Apps enables you to run microservices and containerized applications on a serverless platform. 
+Common use cases include: Deploying API endpoints, Hosting background processing applications and more.
+
+# Azure CLI Login
+First we need to check you are logged in to the Azure in the CLI. The following command will check to see if you are logged in. 
+If not it will open a browser and take you through the login steps.
+
+To Login to Az CLI and select a subscription 
+`az login` followed by `az account list --output table` and `az account set --subscription "name of subscription to use"`
+
+To Install Az CLI
+If you need to install Azure CLI run the following command: curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+
+# Prerequisites
+
+Before you can begin you need to follow the prerequisite steps found here
+
+1. Visit https://github.com/asw101/python-fastapi-pypy
```

Comment:
This is a manual step and thus will not work in an executable doc environment. Exec docs need to be runnable without human interaction.

If the user is expected to grab this content then they should be given the command to grab it automatically and the execution of that command would then be included in unattended execution.